patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -2041,14 +2041,30 @@
  							if (WallID.Sets.Conversion.Grass[wall] && wall != 69) {
  								tile.wall = 69;
  								SquareWallFrame(k, l);
-@@ -39585,6 +_,10 @@
+ 								NetMessage.SendTileSquare(-1, k, l);
+ 							}
+-							else if (TileID.Sets.Conversion.JungleGrass[type] && type != 661) {
```

Comment:
For large changes like this, we like to comment out the original lines with `/*` and `*/` to keep the patches small and easy to update. https://github.com/tModLoader/tModLoader/wiki/tModLoader-Style-Guide#comment-out-vanilla-code-rather-than-removing-it

Basically, comment this out with `/*` and `*/` on new lines. Also add a note to the `/*` line why it is removed, say `/* Moved below since this is a tile conversion not a wall conversion`